package.json

```diff
@@ -50,9 +50,8 @@
     "p-iteration": "^1.1.7"
   },
   "devDependencies": {
-    "bfx-hf-ext-plugin-bitfinex": "^1.0.5",
-    "bfx-hf-models-adapter-lowdb": "^1.0.0",
```

Comment:
dev dep for running the example?

Reply:
yep will add this .. required for running the examples